tests/stdlib/tvarints.nim

```diff
@@ -0,0 +1,84 @@
+discard """
+  cmd:      "nim c -r --styleCheck:hint --panics:on $options $file"
+  matrix:   "-d:danger; -d:release"
+  targets:  "c cpp"
```

Comment:
this is a bug. it'll only run for `nim c` since that's what's in `cmd`.
instead why not simply do:
```
cmd:      "nim $target -r --styleCheck:hint --panics:on $options $file"
```
and drop `targets:  "c cpp"` since CI already runs in both C and CPP

Reply:
=> fixing in https://github.com/nim-lang/Nim/pull/18089